src/pages/matches/[id].tsx

```diff
@@ -151,74 +96,57 @@ const MatchPage = ({ data, errorMessage, errorStatus }: TSsrRole<MatchDto>) => {
           })}
         >
           <Tabs
-            value={tabValue}
+            value={activeTab}
             onChange={handleTabChange}
             aria-label="teams-notes-reports-tab"
             indicatorColor="secondary"
             textColor="inherit"
             variant="fullWidth"
-            centered
           >
-            <Tab label={`${t('NOTES')} (${homeTeamNotes?.totalDocs || 0})`} />
-            <Tab
-              label={`${t('REPORTS')} (${homeTeamReports?.totalDocs || 0})`}
-            />
-            <Tab label={`${t('NOTES')} (${awayTeamNotes?.totalDocs || 0})`} />
-            <Tab
-              label={`${t('REPORTS')} (${awayTeamReports?.totalDocs || 0})`}
-            />
+            {tabs.map(({ name, type, docCount }) => (
+              <Tab
+                key={name}
+                label={`${
+                  type === 'note' ? t('NOTES') : t('REPORTS')
+                } (${docCount})`}
```

Comment:
not some important stuff but can't types be `NOTES` and `REPORTS` to do just that

```suggestion
                label={`${t(type)} (${docCount})`}
```

Reply:
Respectfully disagree - image if someone comes to do some changes here later and for some reason decides to change the `type` name on our object - how is that person supposed to know that it's gonna break translations? Or the other way around - what if someone wants to change translation key for the heading? Is that person supposed to change the `type`? Then the condition breaks.

Long story short - in this case `type` and i18n key are similar only by coincidence and they shouldn't be dependent on each other